CONTRIBUTING.md

```diff
@@ -14,15 +14,44 @@ purposes. As such there are repository "maintainers" who are responsible for
 merging pull requests as well as a "lead maintainer" who is responsible for the
 release cycle, overall merging, moderation and appointment of maintainers.
 
+Getting Started
+---------------
+
+New developers are very welcome and needed. There are a lot of open issues of
```

Comment:
suggest s/a lot/many/

Reply:
It may be a good idea to work in a phrase, similar to this one, early in this section: 

Reviewing and testing is the most effective way you can contribute as a new contributor, and also will teach you much more about the code and process than opening PRs.